external/js/kyber/lib/group/nist/params.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+module.exports = {
+  p256: {
+    name: "P256",
+    bitSize: 256,
+    p: "ffffffffffffffffffffffff00000000000000000000000001000000ffffffff",
```

Comment:
It seems surprising to me that something so fundamental as the p256 parameters would be stored in strings. How do we know they are parsed one single time into the internal format of the BN library?

Reply:
ah, right now every Curve instance will end up creating a new BN object for every curve param. Better to initialize them only once in `params.js`